src/main/java/ulcrs/models/session/Session.java

```diff
@@ -3,52 +3,43 @@
 import ulcrs.models.schedule.Schedule;
 
 import java.time.LocalDateTime;
+import java.time.ZoneOffset;
 
 public class Session {
 
     private String name;
     private Schedule existingSchedule;
-    private LocalDateTime generatedTimestamp;
+    private LocalDateTime createdTimestamp;
     private LocalDateTime lastEditedTimestamp;
 
-    public void save() {
-        // TODO: implement
+    public Session() {
+        this("session_" + LocalDateTime.now().toEpochSecond(ZoneOffset.UTC) + ".json");
```

Comment:
Does this look fine, or do you need a `toString()` call or something similar?

Reply:
It's working.